### ISSUE TITLE
Println breaks array

### ISSUE THREAD

**Issue by d3nzil** [vlang/v#3344]
It's my first time reporting something for V, so let me know if I did something wrong.

**V version:** 
V 0.1.23 eb20dd3
**OS:** 
Mint Linux 19.3 Tricia

**What did you do?**
```v
fn main() {
	mut arr := [1, 2, 3, 4]
	// if this println is commented out, then everything works as expected
	println(arr)
	arr << arr
	println(arr)
}

```
**What did you expect to see?**
[1, 2, 3, 4]
[1, 2, 3, 4, 1, 2, 3, 4]
 
**What did you see instead?**
[1, 2, 3, 4]
[1, 2, 3, 4, 0, 0, 3, 4]

**Comment by alexesprit** [comment]
@d3nzil Can't reproduce the issue on V 0.1.24 c1cc203, Windows 10:
```
$ v run test.v
[1, 2, 3, 4]
[1, 2, 3, 4, 1, 2, 3, 4]
```

Could you update V to the latest version and reproduce the issue again?

**Comment by dhonx** [comment]
https://devbits.app/play/K6GjqpaTVpi9

**Comment by d3nzil** [comment]
I just tried again on Mint linux, with updated V: V 0.1.24 8412c6f
and the problem persists.

**Comment by lutherwenxu** [comment]
Could replicate on ArchLinux.
Code sample returns junk: ```
[1, 2, 3, 4]
[1, 2, 3, 4, 0, 0, 1062375440, 21903]
```

**Comment by alexesprit** [comment]
Fixed in cf1fd6e9500401a78ef0ecd5ce103d630f0b0f5d.
